cpp/src/slice2cpp/Gen.cpp

```diff
@@ -2657,7 +2651,6 @@ Slice::Gen::InterfaceVisitor::visitInterfaceDefEnd(const InterfaceDefPtr& p)
         allOpNames.push_back("ice_isA");
         allOpNames.push_back("ice_ping");
         allOpNames.sort();
-        allOpNames.unique();
```

Comment:
The Slice compiler ensures that there are no duplicate operation names, even among inherited operations.